src/chart/bar/BarView.js

```diff
@@ -336,7 +336,12 @@ var clip = {
         return clipped;
     },
 
-    polar: function (coordSysClipArea) {
+    polar: function (coordSysClipArea, layout) {
+        var r = mathMin(layout.r, coordSysClipArea.r);
+        var r0 = mathMin(layout.r0, coordSysClipArea.r);
+
+        layout.r = r;
+        layout.r0 = r0;
         return false;
```

Comment:
Always returning `false` doesn't seem right. It may be like `var clipped = layout.r < coordSysClipArea.r0 || layout.r0 > coordSysClipArea.r;`

Reply:
And you should also implement the logic at https://github.com/apache/incubator-echarts/pull/12418/files#diff-9f3e66cf4f6fab7ab14223075cd4df3bR42 .